retriever/metrics.go

```diff
@@ -44,7 +44,7 @@ func NewMetrics(httpPort string, logger logging.Logger) *Metrics {
 			},
 		),
 		httpPort: httpPort,
-		logger:   logger,
+		logger:   logger.With("component", "retriever.Metrics"),
```

Comment:
nit: Should we standarize on camel case? So this should be "RetrieverMetrics"?

Reply:
I standardized it to the struct name and prefixed by the `<package_name>.` when the struct name is vague. 
I guess it's cleaner to just camel case it